code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -95,6 +95,16 @@
 		evolve()
 		return
 
+/mob/living/simple_animal/mouse/ClickOn(atom/A, params)
+	if(istype(A, /obj/item/reagent_containers/food/snacks/cheesewedge) && src.canUseTopic(A, BE_CLOSE, NO_DEXTERITY))
```

Comment:
```suggestion
	if(istype(A, /obj/item/reagent_containers/food/snacks/cheesewedge) && canUseTopic(A, BE_CLOSE, NO_DEXTERITY))
```

Reply:
Don't use `src` when it's implied.
Also don't use `canUseTopic`, that's for machine topic. Use `Adjacent()` or the likes.